working-groups/README.md

```diff
@@ -1,7 +1,47 @@
 # Working Groups
 
-Each AMP Working Group is a segment of the community with knowledge/interest in specific area of AMP, with a mandate provided by the Technical Steering Committee.  See [GOVERNANCE.md](../GOVERNANCE.md#working-groups) for more details.
 
-## List of working groups
+An AMP [Working Group](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#working-groups) is a segment of the community with knowledge/interest in specific area of AMP.  Working Groups are created by AMP’s [Technical Steering Committee](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md#technical-steering-committee-tsc). 
+
+AMP’s Working Groups are:
+
+* **Access control and subscriptions.**
+  * Responsible for user specific controlled access to AMP content (amp-subscriptions, amp-access and related extensions)
+  * Facilitator: @jpettitt 
+* **Ads & analytics.**
+  * Responsible for ads & analytics features and integrations in AMP.
+  * Facilitator: @lannka
+* **AMP4Email.**
+  * Responsible for the AMP4Email project.
+  * Facilitator: @choumx
+* **Approvers.**
+  * Responsible for approving changes that have a significant impact on AMP's behavior or significant new features as described in the feature and bug fix process.
+  * Facilitator: @dvoytenko
+* **Code of Conduct.**
+  * Responsible for enforcing AMP's [Code of Conduct](https://github.com/ampproject/meta/blob/master/CODE_OF_CONDUCT.md).  (The Technical Steering Committee delegates its responsibility for enforcement of the Code of Conduct to this Working Group.)
+  * Facilitator: @nainar
+* **Infrastructure.**
+  * Responsible for AMP's infrastructure, including building, testing and release.
+  * Facilitator: @rsimha
+* **Outreach.**
+  * Responsible for helping developers who use AMP remain productive and keeping the AMP community healthy.  This includes maintaining and enhancing AMP's developer-facing sites and documentation, maintaining communication channels, organizing community events, etc.
+  * Facilitator: @pbakaus
+* **Runtime.**
+  * Responsible for AMP's core runtime, such as layout/rendering and data binding.
+  * Facilitator: @jridgewell
+* **Stories.**
+  * Responsible for implementing and improving AMP's story format (amp-story).
+  * Facilitator: @newmuis
+* **UI.**
+  * Responsible for AMP's visual components & interactions.  The UI WG also has overall responsibility for ensuring AMP is accessible, though each WG is responsible for ensuring accessibility within their areas of responsibility. 
+  * Facilitator: @aghassemi
+* **UX.**
+  * Responsible for AMP’s user experience, including published guidelines for AMP UX.
+  * Facilitator: @spacedino
```

Comment:
Needs new fascilitator

Reply:
@cramforce @rudygalfi @aghassemi who should be the facilitator for this WG?

Reply:
That makes sense to me.  @cramforce?

Reply:
SG

Reply:
Alright, I've moved these responsibilities into the UI WG for now.